vervet-underground/internal/storage/gcs/client_test.go

```diff
@@ -0,0 +1,157 @@
+package gcs_test
+
+import (
+	"bytes"
+	"os"
+	"strconv"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/rs/zerolog/log"
+
+	"vervet-underground/internal/storage"
+	"vervet-underground/internal/storage/gcs"
+)
+
+const (
+	accessKey   = "test"
+	secretKey   = "test"
+	sessionKey  = "test"
+	gcsEndpoint = "http://0.0.0.0:4443/storage/v1/"
+	gcsRegion   = "US-CENTRAL1" // https://cloud.google.com/storage/docs/locations#location-r
+	projectId   = "test"
+)
+
+var cfg = &gcs.Config{
+	GcsRegion:   gcsRegion,
+	GcsEndpoint: gcsEndpoint,
+	Credentials: gcs.StaticKeyCredentials{
+		AccessKey:  accessKey,
+		SecretKey:  secretKey,
+		SessionKey: sessionKey,
+		ProjectId:  projectId,
+	},
+}
+
+func cleanup() {
+	// cleanup
+	client, err := gcs.New(cfg)
+	if err != nil {
+		log.Error().Err(err).Msg("failed to initialize GCS storage")
+		return
+	}
+	st, ok := client.(*gcs.Storage)
+	if !ok {
+		log.Error().Err(err).Msg("failed to cast to GCS storage")
```

Comment:
I don't think there's an `err` here to log.
```suggestion
		log.Error().Msg("failed to cast to GCS storage")
```